episodes/infrastructure.md

```diff
@@ -196,12 +196,16 @@ You may need to wait a few minutes for the website to be generated.
 
 The lesson title can be adjusted by modifying the `config.yaml` file in the repository.
 The `config.yaml` file contains several global parameters for a lesson -
-to determine some of the page styling, contact details for the lesson, etc -
-and is written in _[YAML]_, a structured file format of key-pair values.
+to determine some of the page styling, contact details for the lesson, etc. 
+`config.yaml` is written in _[YAML]_, a structured file format of key-value pairs in the form `key: value`.
 As well as the title of the lesson,
 you can and should adjust some of the other values in `config.yaml`,
 but you should not need to add new values or learn a lot about YAML
-to be able to configure your lesson.
+to be able to configure your lesson. You should also only have to add or modify single-line 
+string values and not deal with multi-line strings or other data types (e.g. numbers, booleans, lists, dictionaries) 
+that YAML supports. YAML generally does not require quotes around string values unless 
+they contain [reserved characters](https://www.w3schools.io/file/yaml-escape-characters/). It may also be good practice 
+from the readibility point of view to use quotes if your string contains several words.
```

Comment:
```suggestion
they contain [reserved characters](https://www.w3schools.io/file/yaml-escape-characters/) though it may be good practice 
from readability point of view to use quotes if your string contains several words.
```